backend/kesaseteli/common/utils.py

```diff
@@ -118,6 +122,23 @@ def validate_finnish_social_security_number(value):
         )
 
 
+def get_random_social_security_number_for_year(year: int) -> str:
```

Comment:
Perhaps this util function could be under tests folder if it's only used in tests?

Reply:
Taken into account